candidate_word.h

```diff
@@ -20,9 +20,7 @@
 #ifndef __CANDIDATE_WORD_H__
 #define __CANDIDATE_WORD_H__
 
-#include <QDeclarativeView>
-#include <QObject>
-#include <qdeclarative.h>
```

Comment:
Ok, looking around, I see that it seems to make some projects compile a lot slower. I'll fix this, thanks.